### ISSUE TITLE
Support using 'aria-valuenow', 'aria-valuemin' and 'aria-valuemax' on a focusable elements with role separator

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#552]
Currently the 'aria-allowed-attr' rule is failing if we use 'aria-valuenow' on a focusable element with role separator

Here is the Aria 1.1 spec for role separator
![image](https://user-images.githubusercontent.com/22068434/30987605-7989b2a2-a44c-11e7-84c5-43ef24f79bc4.png)

We need to add support for:
- Using 'aria-valuenow', 'aria-valuemin' and 'aria-valuemax' on a focusable element with role separator
- Failing the 'aria-required-attr' rule when a focusable element does not contain these attributes.

**Comment by WilcoFiers** [comment]
The reason we haven't added `aria-valuenow` is because last we test support for this is still lacking. This attribute is used to convey important information about a control. Without sufficient support, it's likely users of AT won't be able to use successfully.

Have you looked at how well this feature you are building is supported in assistive technologies? If there's sufficient support, or if we can find a way to identify alternative solutions we could add it.

**Comment by isner** [comment]
+1 to @WilcoFiers . I just conducted some testing with Safari/VoiceOver and Firefox/NVDA, and neither recognized the values (min, max, now) on a focusable separator.

**Comment by WilcoFiers** [comment]
@iamrafan I don't think this should be supported by axe-core just yet, since AT don't support this. Unless you have any other suggestions I'm going to close this for now.

**Comment by iamrafan** [comment]
@isner : Even I found it does not work with chrome/NVDA either. 
@WilcoFiers: I am fine with closing this issue for now.
